mobile-widgets/qmlmanager.cpp

```diff
@@ -540,6 +542,8 @@ void QMLManager::consumeFinishedLoad(timestamp_t currentDiveTimestamp)
 	if (informational_prefs.unit_system == IMPERIAL)
 		informational_prefs.units = IMPERIAL_units;
 	prefs.units = informational_prefs.units;
+	prefs.tankbar = git_prefs.tankbar;
+	prefs.dcceiling = git_prefs.dcceiling;
```

Comment:
I need to study the code some more to figure out why we do this twice. That seems redundant, but a quick read through the code wasn't quite enough to figure out what's going on there.